docs/users/step-debugging.md

```diff
@@ -1,4 +1,4 @@
-# Step-debugging with ddev and xdebug
+<h1>Step-debugging with ddev and xdebug</h1>
```

Comment:
This is the simplest way to provide a page title without having it display as a heading in the sidebar. Markdown headings seem to populate the sidebar automatically.